pkg/splunk/client/minioclient.go

```diff
@@ -138,6 +140,29 @@ func (client *MinioClient) GetAppsList() (S3Response, error) {
 	return s3Resp, nil
 }
 
+// DownloadApp downloads an app package from remote storage
+func (client *MinioClient) DownloadApp(remoteFile string, localFile string) (bool, error) {
+	scopedLog := log.WithName("DownloadApp")
+
+	file, err := os.Create(localFile)
+	if err != nil {
```

Comment:
Will this fail if a file already exists?

Reply:
Nope, it does not